app/components/createService/createservice.html

```diff
@@ -445,6 +446,52 @@
           <!-- tab-placement -->
           <div class="tab-pane" id="placement" ng-include="'app/components/createService/includes/placement.html'"></div>
           <!-- !tab-placement -->
+          <!-- tab-resources -->
+          <div class="tab-pane" id="resources">
+            <form class="form-horizontal" style="margin-top: 15px;">
+              <div class="form-group">
+                <!-- cpu-limit-input -->
+                <label for="cpu-limit" class="col-sm-2 col-lg-2 control-label text-left">CPU limit<portainer-tooltip position="top" message="Maximum number of CPU the service can use (0 to unlimited)."></portainer-tooltip></label>
+                <div class="col-sm-4 col-lg-4">
+                  <input type="number" step="0.1" class="form-control" ng-model="formValues.CpuLimit" id="cpu-limit" placeholder="e.g. 0.5">
+                </div>
+                <!-- !cpu-limit-input -->
+                <!-- cpu-reservation-input -->
+                <label for="cpu-reservation" class="col-sm-2 col-lg-2 control-label text-left">CPU res<portainer-tooltip position="top" message="Minimum number of CPU guaranteed to the service."></portainer-tooltip></label>
+                <div class="col-sm-4 col-lg-4">
+                  <input type="number" step="0.1" class="form-control" ng-model="formValues.CpuReservation" id="cpu-limit" placeholder="e.g. 0.3">
+                </div>
+                <!-- !cpu-reservation-input -->
+              </div>
+              <div class="form-group">
+                <!-- memory-limit-input -->
+                <label for="memory-limit" class="col-sm-2 col-lg-2 control-label text-left">Mem limit<portainer-tooltip position="top" message="Maximum memory the service can use (0 to unlimited)."></portainer-tooltip></label>
+                <div class="col-sm-2 col-lg-2">
+                  <input type="number" step="0.125" class="form-control" ng-model="formValues.MemoryLimit" id="memory-limit" placeholder="e.g. 128">
+                </div>
+                <div class="col-sm-2 col-lg-2">
+                  <select class="form-control" ng-model="formValues.MemoryLimitUnit">
+                    <option value="MB">MB</option>
+                    <option value="GB">GB</option>
+                  </select>
+                </div>
+                <!-- !memory-limit-input -->
+                <!-- memory-reservation-input -->
+                <label for="memory-reservation" class="col-sm-2 col-lg-2 control-label text-left">Mem res<portainer-tooltip position="top" message="Minimum memory guaranteed to the service."></portainer-tooltip></label>
```

Comment:
Please replace the `Mem res` label text with `Memory reservation`.

Also, please replace the tooltip message with: `Minimum memory available on a node to run a task`